astropy/coordinates/__init__.py

```diff
@@ -21,3 +21,22 @@
 from .funcs import *
 from .calculation import *
 from .solar_system import *
+
+
+from astropy import config as _config
+
+
+class Conf(_config.ConfigNamespace):
+    """
+    Configuration parameters for `astropy.coordinates`.
+    """
+
+    skycoord_init_counter_warn_threshold = _config.ConfigItem(
+        100,
```

Comment:
I would have thought this would be much higher, more like 10000 or 100000.  I see a time of 260 usec / init, so that means 2.6 to 26 seconds in SkyCoord before raising a flag.

Reply:
FWIW I would personally lean towards ~10000 or so